src/tests/cxxfrontend.c

```diff
@@ -54,7 +54,7 @@
 #include "template.h"
 #include "tokens.h"
 #include "version.h"
-#include "visitor.h"
+#include "visitors/package.h"
```

Comment:
Can this just be kept in toplevel?  I'd be ok with a rename to `visitors.h`

Reply:
Sure, I wasn't fully sure about this anyways (that's why I kept this change as a separate commit).